### PR TITLE
Update checksums for R 3.6.0 packages and add patch for ppc

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/R/R-3.6.0-fosscuda-2019a.eb [hunk 1]
```diff
@@ -2281,6 +2284,9 @@ exts_list = [
     ('HiddenMarkov', '1.8-11', {
         'checksums': ['4a1614249eee9f428bc182ea9ced443dff4eafa7babf4259c720e5b4da2d08fa'],
     }),
+    ('HiddenMarkov', '1.8-11', {
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.6.0-fosscuda-2019a.eb:2287]: @Flamefire Looks like you accidentally copied the entry above here?

Comment (Flamefire) [reply easybuild/easyconfigs/r/R/R-3.6.0-fosscuda-2019a.eb]: Thanks. Fixed up the commit.

**Comment by Flamefire** [comment]
I also added the patch for liquidSVM which makes it build on PPC (hope it still applies)

**Comment by boegel** [comment]
@Flamefire Is there a test report coming for ppc?

**Comment by Flamefire** [comment]
Makes sense, yes. Just submitted a job to test that. Gonna take a whole though, so maybe in 10 hrs :/

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3300.joltik.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/362457742f53521870b101eb6518ce42 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2612.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/9d1242b6122202e25831157ada95b141 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3411.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/7b1ab67eee8348d048b4227c03e58dc4 for a full test report.

**Comment by boegel** [comment]
Looking good on x86_64... Any word on a test report from PPC @Flamefire?

If it's too much trouble I'll just go ahead and merge this, I trust you did the right thing there (and it's the same patch for `liquidSVM` we already have in other `R` easyconfigs)

**Comment by Flamefire** [comment]
I have no idea what went wrong with the job, I assumed it would have already submitted here but now I can't even see the job on the cluster anymore. Restarting...

**Comment by Flamefire** [comment]
Test report by @Flamefire
**FAILED**
Build succeeded for 6 out of 8 (2 easyconfigs in this PR)
taurusml3 - Linux RHEL 7.6, POWER, 8335-GTX, Python 2.7.5
See https://gist.github.com/b9714f54eebeb7d207b2446663219067 for a full test report.

**Comment by Flamefire** [comment]
I'm now running into the same issue I mentioned earlier: Installing `viridis` fails (likely) due to `viridisLite` being installed first. No idea how to solve as I blessfully know nothing about R and its installation ;) Maybe there is something else somewhere in the environment which R picks up? No idea how to check that

Message I get is:
> 'viridis.map' is not an exported object from 'namespace:viridisLite'

The "owner" of viridis says https://github.com/sjmgarnier/viridis/issues/57#issuecomment-381570140
>  install viridis and viridis only. Do not install viridisLite first.

But it seems viridis would otherwise download and install viridisLite first, so we need to...

**Comment by boegel** [comment]
@Flamefire I'm clueless on that too... Why doesn't that issue occur on x86_64?!

**Comment by Flamefire** [comment]
As I have no idea why it appears I don't know why it doesn't appear either. I tried to install it on Power till it fails and wanted to manually continue or retry with `--skip` but that doesn't seem to work: It reinstalls everything. If I could get a shell in the environment where it tries to install viridis at that point as EB would, I could maybe find out more. But as it seems our admins could install it successfully maybe we shouldn't bother...

**Comment by boegel** [comment]
Since the test installations are working out fine n `x86_64`, I'm calling this: time to merge.

Thanks for the effort @Flamefire!

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @Flamefire!

**Comment by Flamefire** [comment]
Test report by @Flamefire
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
taurusi4104.taurus.hrsk.tu-dresden.de - Linux RHEL 7.8, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/650ea4bdd1f171b97a02c9ab5fb423ed for a full test report.

**Comment by Flamefire** [comment]
Ok problem found: On Power we have a Lustre mounted over NFS and apparently the install creates and writes to a file but a subsequent `getdents` on the folder the file was created (and `chmod`ed) in does not return it.
Workaround: Install in /tmp or similar for testing

Relevant part of strace:
```
[pid 74646] open("/software/R/3.6.0-fosscuda-2019a/lib64/R/library/00LOCK-viridisLite/00new/viridisLite/data/viridis.map.rda", O_WRONLY|O_CREAT|O_TRUNC, 0666) = 4
[pid 74646] fstat(3, {st_mode=S_IFREG|0644, st_size=27832, ...}) = 0
[pid 74646] mmap(NULL, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x2002cb230000
[pid 74646] read(3, "BZh91AY&SY>\232Q\6\0,\205\177\377\377\377\377\377\377\377\377\377\377\377\377\377\377"..., 8192) = 8192
[pid 74646] fstat(4, {st_mode=S_IFREG|0644, st_size=0, ...}) = 0
[pid 74646] mmap(NULL, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x2002cb240000
[pid 74646] write(4, "BZh91AY&SY>\232Q\6\0,\205\177\377\377\377\377\377\377\377\377\377\377\377\377\377\377"..., 8192) = 8192
[pid 74646] read(3, "L\352O\317\317\354\321\33\343mq\224\330\213<\317$1U\33+4z\377\301<\307ZSN\245\306"..., 8192) = 8192
[pid 74646] read(3, "?\300v\360\374\243(\205\3068\375\376\274J\335\3729\217T\250a)\226\340\325M_\275/\340\337^"..., 8192) = 8192
[pid 74646] write(4, "L\352O\317\317\354\321\33\343mq\224\330\213<\317$1U\33+4z\377\301<\307ZSN\245\306"..., 8192) = 8192
[pid 74646] write(4, "?\300v\360\374\243(\205\3068\375\376\274J\335\3729\217T\250a)\226\340\325M_\275/\340\337^"..., 8192) = 8192
[pid 74646] read(3, "\224W\2023\303\310\365\341\2412\365\370\31\251\327\200\201\377-\273`M-\235\331\231,\n\341\310L\21"..., 8192) = 3256
[pid 74646] read(3, "", 8192)           = 0
[pid 74646] write(4, "\224W\2023\303\310\365\341\2412\365\370\31\251\327\200\201\377-\273`M-\235\331\231,\n\341\310L\21"..., 3256) = 3256
[pid 74646] close(4)                    = 0
[pid 74646] munmap(0x2002cb240000, 8192) = 0
[pid 74646] chmod("/software/R/3.6.0-fosscuda-2019a/lib64/R/library/00LOCK-viridisLite/00new/viridisLite/data/viridis.map.rda", 0644) = 0
[pid 74646] close(3)                    = 0
[pid 74646] munmap(0x2002cb230000, 8192) = 0
[pid 74646] openat(AT_FDCWD, "/software/R/3.6.0-fosscuda-2019a/lib64/R/library/00LOCK-viridisLite/00new/viridisLite/data", O_RDONLY|O_NONBLOCK|O_DIRECTORY|O_CLOEXEC) = 3
[pid 74646] getdents(3, /* 2 entries */, 32768) = 48
[pid 74646] getdents(3, /* 0 entries */, 32768) = 0
```

The 2nd last line should show "3 entries" as it does on x86 or /tmp

**Pull request merged**